File: Modul 5/Python/PRAK502.py
def hitung(nilai1, nilai2):
    hasil_hitung = nilai1 - nilai2 
    if (hasil_hitung < 0):
        result = -hasil_hitung
    else:
        result = hasil_hitung
    return result

def mutlak(angka):
    if angka < 0:
        return -angka
    else:
        return angka

angka = input().split()
a, c, b, d = map(int,angka)
hasil = hitung(a, b) + hitung(c, d)
print(mutlak(hasil))

File: Modul 5/C/PRAK502.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2){
//Lengkapi Function ini
    int hasil_hitung = nilai1 - nilai2;
    if (hasil_hitung < 0) {
        return -hasil_hitung;
    } else {
        return hasil_hitung;
    }
}

int mutlak(int angka) {
//Lengkapi Function ini
    if (angka < 0) {
        return -angka;
    } else {
        return angka;
    }
}

int main() {
    int a, b, c, d;
    scanf("%d", &a);
    scanf("%d", &c);
    scanf("%d", &b);
    scanf("%d", &d);
    int Hasil = hitung(a, b) + hitung(c, d);
    printf("%d", mutlak(Hasil));
    return 0;
}